front/app/components/Avatar/index.tsx

```diff
@@ -124,104 +120,108 @@ export interface Props {
   hideIfNoAvatar?: boolean;
 }
 
-const Avatar = memo(
-  ({
-    isLinkToProfile,
-    moderator,
-    className,
-    addVerificationBadge,
-    userId,
-    hideIfNoAvatar,
-    ...props
-  }: Props & WrappedComponentProps) => {
-    const user = useUser({ userId });
-
-    if (!isNilOrError(user)) {
-      const { slug, avatar, verified } = user.attributes;
-      const profileLink = `/profile/${slug}`;
-      // In dev mode, slug is sometimes undefined,
-      // while !isNilOrError(user) passes... To be solved properly
-      const hasValidProfileLink = profileLink !== '/profile/undefined';
-      const avatarSize = props.size;
-      const paddingValue = props.padding || 3;
-      const borderThickness = props.borderThickness || 1;
-      const hasHoverEffect = (isLinkToProfile && hasValidProfileLink) || false;
-      const imageSizeLabel = avatarSize > 160 ? 'large' : 'medium';
-      const avatarSrc = avatar ? avatar[imageSizeLabel] : null;
-      const containerSize = avatarSize + paddingValue * 2 + borderThickness * 2;
-      const badgeSize = avatarSize / (avatarSize < 40 ? 1.8 : 2.3);
-      const fillColor = props.fillColor || lighten(0.2, colors.textSecondary);
-      const fillHoverColor = colors.textSecondary;
-      const borderHoverColor = colors.textSecondary;
-      const borderColor = props.borderColor || 'transparent';
-      const bgColor = props.bgColor || 'transparent';
-
-      if (!avatarSrc && hideIfNoAvatar) {
-        return null;
-      }
-
-      const AvatarComponent = (
```

Comment:
Yeah that's strange! Changes look fine to me though :) There aren't any tests we could add for this are there? Or maybe we can bring this up at the next FE meeting so other devs are aware as well?

Reply:
> There aren't any tests we could add for this are there? 
I'm not sure, weglot is making requests to an external api and then makes direct changes to the DOM... could in theory be tested in an e2e test but I'm afraid it would be very flaky

> Or maybe we can bring this up at the next FE meeting so other devs are aware as well?
Sure, I'll add it to the agenda!